recipes/7zip/19.00/conanfile.py

```diff
@@ -11,12 +11,12 @@ class Package7Zip(ConanFile):
     license = ("LGPL-2.1", "BSD-3-Clause", "Unrar")
     homepage = "https://www.7-zip.org"
     topics = ("conan", "7zip", "zip", "compression", "decompression")
-    settings = "os_build", "arch_build", "compiler"
+    settings = "os", "arch", "compiler"
```

Comment:
Shouldn't `build_type` be added as well?

Reply:
Hm, yeah, maybe? I guess `vcvars()` wouldn't automatically add `/Zi` or similar, so we would still have to do that manually, right?

Reply:
The problem with not adding `build_type` is that the build machine can choose whatever build_type it pleases.

Reply:
Sure, but that _shouldn't_ affect the build unless we actually use the setting, right?

Reply:
If the build profile has `build_type=Release`, then it should be fine.

Reply:
What I mean is, if we don't add `build_type` to `settings`, shouldn't Conan just ignore the attribute and use the same build regardless of its value at build?

Reply:
If I build it with Debug, then the package_id will be the same as when you build it with Release.
conan will assume that Release == Debug